docs/semgrep-app/getting-started-with-semgrep-app.md

```diff
@@ -99,17 +89,113 @@ Scanning is Semgrep's primary operation. When you first sign into Semgrep App, i
+
+:::note
+Scans from local repositories do not access their corresponding remote repositories. For this reason, links to specific lines of code in the Findings page are not created. See [Linking local scans to their remote repositories](#linking-local-scans-to-their-remote-repositories) for a workaround.
+:::
+
+##### Linking local scans to their remote repositories 
+
+![Screenshot of findings page snippet with no hyperlinks](../img/findings-no-hyperlinks.png "Screenshot of findings page snippet with no hyperlinks")
+*Figure 1.* Partial screenshot of findings page with no hyperlinks.
+
+Local repository scans require additional configuration to link to their corresponding remote repositories. Set up environment variables within your command line to configure cross-linking between local and remote repositories within Semgrep App.
+
+To set up environment variables:
+
+1. Ensure that your current working directory is the root of the repository to create links for. 
+2. Set up the `SEMGREP_REPO_URL`:
+    1. Retrieve the URL by navigating to your online repository. Copy the value in the address bar. This is your `URL_ADDRESS`.
+    2. Set the variable by entering the text below, substituting `URL_ADDRESS` with the value from the previous step.
+    <pre><code>
+    SEMGREP_REPO_URL=<em>URL_ADDRESS</em>
+    </code></pre>
+3. Set up the `SEMGREP_BRANCH`:
+    1. Run the following to retrieve the branch name:
+    ```bash
+        git rev-parse --abbrev-ref HEAD
+    ```
+    2. Set the variable by entering the text below, substituting `BRANCH_NAME` with the value from the previous step.
+    <pre><code>
+    SEMGREP_BRANCH=<em>BRANCH_NAME</em>
+    </code></pre>
+4. Set up the `SEMGREP_REPO_NAME`:
+    1. Retrieve the repository name by logging in to your GitHub or GitLab account and copying the repository name from your dashboard. 
+    2. Set the variable by entering the text below, substituting `REPO_NAME` with the value from the previous step.
+    <pre><code>
+    SEMGREP_REPO_NAME=<em>REPO_NAME</em>
+    </code></pre>
+5. Set up the `SEMGREP_COMMIT`:
+    1. Run the following to retrieve the commit hash:
+    ```bash
+        git log -n 1
+    ```
+    2. Set the variable by entering the text below, substituting `COMMIT_HASH` with the value from the previous step.
+    <pre><code>
+    SEMGREP_COMMIT=<em>COMMIT_HASH</em>
+    </code></pre>
+
+Sample values:
+
+```
+# Set the repository URL
+$> SEMGREP_REPO_URL=https://github.com/corporation/s_juiceshop
+
+# Set the repository name
+$> SEMGREP_REPO_NAME=corporation/s_juiceshop
+
+# Retrieve the branch 
+$> git rev-parse --abbrev-ref HEAD
+s_update
+# Set the branch
+$> SEMGREP_BRANCH=s_update
+
+# Retrieve the commit hash
+$> git log -n 1
+commit fa4e36b9369e5b039bh2220b5h9R61a38b077f29 (HEAD -> s_juiceshop, origin/master, origin/HEAD, master)
+# Set the commit hash
+$> SEMGREP_COMMIT=fa4e36b9369e5b039bh2220b5h9R61a38b077f29

```

Comment:
Can we anonymize the commit hash, even if it's just changing 1 or 2 alpha numericals?